### PR TITLE
activated state of switches match textsecure color dark

### LINKED ISSUE #3881: Switchcompat sliders in dark theme to match the ones in light theme
In different preferences sliders (switchcompat?) are used.  In the light theme, they are the TS blue which looks great, while in the dark theme they are aqua.  While this is certainly not a show-stopper, it would look nicer if they matched the same blue or a more muted blue than having them be aqua.  For someone who knows Android dev, unlike myself, this is probably pretty easy.

### PR DESCRIPTION
**Pull request by heyimgay**
fixes #3881

I set colorControlActivated to @color/textsecure_primary_dark

should the same be done with the light theme - setting the activated state to the textsecure color?

### CODE REVIEW

**Comment by heyimgay** [comment]
so this accidentally became two commits - is there an easy way to merge them? or how do i remove them and redo  them?

@mcginty also i can help triaging/tagging bugs

**Comment by jeremymasters** [comment]
I think this is what you might be looking for to squash commits: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html.

**Comment by heyimgay** [comment]
Thanks!

**Comment by jeremymasters** [comment]
Not to try to push too many things into one, but is it as simple to change the typing cursor when editing/typing in fields in the dark theme to the same dark theme blue? instead of the aqua?

File: res/values/themes.xml [hunk 1]
```diff
@@ -188,6 +188,7 @@
         <item name="actionBarPopupTheme">@style/ThemeOverlay.AppCompat.Dark</item>
         <item name="android:textColor">@color/text_color_dark_theme</item>
         <item name="android:textColorSecondary">@color/text_color_secondary_dark_theme</item>
+        <item name="colorControlActivated">@color/textsecure_primary_dark"</item>
```
Comment (mcginty) [inline res/values/themes.xml]: please make sure your diffs compile (notice the `"` at the end after dark)

**Comment by mcginty** [comment]
@jeremymasters turns out it is.

Merging the corrected version of this PR as well as another one I'm piling on top to add the missing `colorControl` variables to set the cursor colors correctly for EditTexts as well. In 3.5.0

**Comment by jeremymasters** [comment]
@mcginty awesome!

**Pull request closed without merging**